specs/UrbanMobility/GtfsShape/schema.json

```diff
@@ -24,28 +26,14 @@
           "minItems": 1
         },
         "location": {
-           "type": "object",
-           "oneOf": [{
-              "properties": {
-                "type": {
-                  "type": "string",
-                  "enum": ["LineString"]
-                },
-                "coordinates": { "$ref": "http://json-schema.org/geojson/geometry.json#/definitions/lineString" }
-              }
+          "oneOf": [
+            {
+              "$ref": "http://geojson.org/schema/Geometry.json#LineString"
             },
             {
-                "properties": {
-                  "type": {
-                    "type": "string",
-                    "enum": ["MultiLineString"]
-                  },
-                  "coordinates": {
-                    "type": "array",
-                    "items": { "$ref": "http://json-schema.org/geojson/geometry.json#/definitions/lineString" }
-                  }
-                }
-           }]
+              "$ref": "http://geojson.org/schema/Geometry.json#MultiLineString"
```

Comment:
this cannot be referenced using actual url

Reply:
It depends on how you map that URL to an actual schema. I don't like to have individual schemas per type of GeoJSON Geometry I think it does not make sense

Reply:
URL is by definition something that you resolve :) So while I understand your point, I disagree to the "hack". The solution is getting approved the change in GeoJSON and meanwhile adhere to what they offer.

Reply:
well, they are URIs not URLs ... https://json-schema.org/latest/json-schema-core.html#rfc.section.8.2 so not necessarily dereferenceable

Reply:
look, I am not stubborn on this, I just believe that not creating misalignments is the good way to go :)